src/com/facebook/buck/cxx/CxxLibraryDescription.java

```diff
@@ -1226,6 +1226,7 @@ public CxxPlatform getDefaultCxxPlatform() {
     // used otherwise.
     public Optional<SourcePath> bridgingHeader;
     public Optional<String> moduleName;
+    public Optional<String> swiftProcessScript;
```

Comment:
It should be in both `SwiftLibraryDescription ` and `CxxLibraryDescription`.

Following the way it swift logic is built, we need to pass this from `apple_library`, so we need to add this into `CxxLibraryDescription`.
Then `CxxLibraryDescription` will pass this information to `SwiftLibraryDescription`, thus we also need to add this into `SwiftLibraryDescription`.